### ISSUE TITLE
Question: correct way to load weights from post-training quantized model into a new model

### ISSUE THREAD

**Issue by amad-person** [NervanaSystems/distiller#480]
I wanted to collect the gradient norms for my training/testing data by "training" my post-training quantized ResNet20_CIFAR model for one epoch. 

But since my quantized model is using the `RangeLinearQuantParamLayerWrapper` for simulating post-training quantization, I can't run the model in `eval()` mode. I've been following the related #185 issue, but I was wondering if there is a temporary workaround I could use. 

Specifically, is there a way I can just copy the simulated integer weights into a vanilla ResNet20_CIFAR model? Since the state_dicts for vanilla and post-training quantized model aren't the same I wanted to know the correct weights to copy over. Thanks!

**Comment by guyjacob** [comment]
Hi @amad-person,

Indeed post-training quantized modules weren't meant to be used in training. You could, if you really wanted, comment-out the eval-only check here:
https://github.com/NervanaSystems/distiller/blob/2291fdcc2ea642a98d4e20629acb5a9e2e04b4e6/distiller/quantization/range_linear.py#L550-L552
But even if you did that and were actually able to get the gradients (that is - if it doesn't raise an error elsewhere for whatever reason), I'd expect those gradients to be 0 almost everywhere, because the `round` function is used during quantization.

Regarding getting the integer weights from the state_dict - if the original layer is named `conv1`, for example, then in the post-training quantized model you'll find the weights for that layer at `conv1.wrapped_module.weight`.

Hopefully this helps, though I admit I'm not totally sure what you're trying to do :)
